graphics/partio/Portfile

```diff
@@ -0,0 +1,32 @@
+PortSystem		1.0
+PortGroup		github 1.0
+PortGroup		cmake 1.1
+
+name			partio
+version			1.9.1
+categories		graphics
+platforms		darwin
+license			BSD
+maintainers		nomaintainer
+homepage		http://partio.us
+description		C++ library for manipulating a variety of \
+			particle formats
+long_description 	Partio is an open-source C++ library, \
+			developed by Walt Disney Animation Studios, \
+			for reading, writing, and manipulating a \
+			variety of standard particle formats (GEO, \
+			BGEO, PTC, PDB, PDA). It allows software \
+			applications to read and write point cloud \
+			formats from a variety of computer graphics \
+			programs and can serve as a way to interchange \
+			particle data between various file formats.
+
+github.setup		wdas partio ${version} v
```

Comment:
The `github.setup` line should be moved to near the top of the Portfile, just above the current `name` line. Then the `name` and `version` lines become redundant and should be removed.

Reply:
I've made the requested changes.